http/preseed.cfg

```diff
@@ -0,0 +1,38 @@
+d-i passwd/root-login boolean true
```

Comment:
The goal is just to get to a bare bones image we can run ansible against before the final commit.